src/models/newJobHandler.ts

```diff
@@ -31,6 +33,32 @@ export class NewJobHandler implements IJobHandler<IngestionJobParams> {
     }
   }
 
+  private async createPolygonParts(
+    job: IJobResponse<IngestionJobParams, unknown>,
+    validatedRequestBody: PolygonPartsPayload
+  ): Promise<PolygonPartsEntityNameObject> {
+    try {
+      return await this.polygonPartsManager.createPolygonParts(validatedRequestBody);
+    } catch (error) {
+      if (error instanceof ConflictError) {
```

Comment:
its leads to instant error from `getExistingPolygonPartsEntity` because "polygonPartsEntityName" will be never exists if there an conflict error

Reply:
we should exposes entity exists check by polygon-parts-manager service and use it here before update the job parameter by its response value.

Reply:
we talked